bindings/Modules/tests/CMakeLists.txt

```diff
@@ -14,21 +14,23 @@ find_package(SofaFramework REQUIRED)
 
 enable_testing()
 add_executable(${PROJECT_NAME} ${SOURCE_FILES} ${PYTHON_FILES})
-target_link_libraries(${PROJECT_NAME} SofaGTestMain SofaHelper SofaPython3::Plugin SofaPython3::Bindings.Modules)
+target_link_libraries(${PROJECT_NAME} SofaGTestMain SofaHelper SofaPython3::Plugin)
 target_compile_definitions(${PROJECT_NAME} PRIVATE "PYTHON_TESTFILES_DIR=\"${CMAKE_CURRENT_SOURCE_DIR}/\"")
 set_target_properties(${PROJECT_NAME} PROPERTIES FOLDER "Bindings/Tests")
 
-set(RPATH "$ORIGIN/../lib")
+# This will set rpaths relative to all SOFA core AND relocatable dependencies
+sofa_auto_set_target_rpath(
+    TARGETS ${PROJECT_NAME}
+    RELOCATABLE "plugins"
+)
+
+# This will set rpaths relative to SP3 plugin library
+get_target_property(RPATH ${PROJECT_NAME} "INSTALL_RPATH")
+list(APPEND RPATH "$ORIGIN/../lib")
 if (APPLE)
     list(APPEND RPATH "@executable_path/../lib")
 endif()
-
-set_target_properties(
-        ${PROJECT_NAME}
-        PROPERTIES
-        INSTALL_RPATH_USE_LINK_PATH TRUE
-        INSTALL_RPATH "${RPATH}"
-)
+set_target_properties(${PROJECT_NAME} PROPERTIES INSTALL_RPATH "${RPATH}")
```

Comment:
Instead of repeating this in all bindings, I think you could do it once in bindings/CMakeLists.txt and `sofa_auto_set_target_rpath` will not override it.
```cmake
# RPATH
if(UNIX)
    set(CMAKE_INSTALL_RPATH
        "$ORIGIN/../lib"
        "$$ORIGIN/../lib"
        )
    if(APPLE)
        list(APPEND CMAKE_INSTALL_RPATH
            "@loader_path/../lib"
            "@executable_path/../lib"
            )
    endif()
endif(UNIX)
```